docs/Contributing/API-for-contributors.md

```diff
@@ -3184,6 +3185,73 @@ Notifies the server about an agent error, resulting in two outcomes:
 
 `Status: 500`
 
+### Register a device with the setup experience
```

Comment:
Hey @dantecatalfamo, @rachaelshaw and I are looking at this endpoint and we're wondering if we still need it.

IIRC there was some discussion about using some event on the device or some MDM command response to trigger the setup experience.

Reply:
Hey @noahtalerman,

I was discussing this with @roperzh and @jahzielv this morning. It looks like there's way to identify the beginning of a new DEP installations without the need for this endpoint. It's not necessary anymore, details here https://github.com/fleetdm/fleet/issues/22379

Reply:
cc @marko-lisica

Reply:
Seems that we implemented Orbit-authenticated endpoints:

`POST /api/fleet/orbit/setup_experience/start` & `POST /api/fleet/orbit/setup_experience/status`

Could someone help me change the docs to reflect what has been implemented? @jahzielv @dantecatalfamo?

cc @noahtalerman